content/dart/concepts/map/terms/length/length.md

```diff
@@ -0,0 +1,77 @@
+---
+Title: 'length'
+Description: 'Retrieves the number of key-value pairs in a Map.'
+Subjects: 
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Dart'
+  - 'Properties'
+  - 'Map'
+CatalogContent:
+  - 'learn-dart'
+  - 'paths/computer-science'
+---
+
+In Dart, the **`.length`** property is a utility for determining the number of key-value pairs contained within a Map instance. It enables developers to efficiently manage and manipulate data structures by providing a quick and reliable means of accessing the size of a Map.
+
+## Syntax
+
+```pseudo
+print(<Map-variable>.length);
+```
+
+- `print();`: A built-in function that outputs data to the console.
+- `<Map-variable>`: Represents a variable that holds a Map object.
+
+## Example 1
+
+In the following example, the `.length` property is used to get the key-value pairs of a Map instance that uses similar DataTypes.
+
+```dart
+Map<String, String> countries = {
+    'USA': 'United States',
+    'IRE': 'Ireland',
+    'ITLY': 'Italy'
+};
+print(countries.length);
+```
+Here is the output for the example above:
+
+```shell
+3
+```
+
+## Example 2
+
+The `.length` property can take different DataTypes for its key-value pairs as well.
+
+```dart
+Map<String, int> fruits = {
+    'Apple': 3,
+    'Banana': 2,
+    'Mango': 9
+}
```

Comment:
The print statement is not there so there would be no output. Please add the print statement so it'll be a valid code and output block.